lib/relay.js

```diff
@@ -6,6 +6,7 @@ const uuid = require('uuid/v4');
 const Filters = require('./filters');
 const replace = require('./replace-vars');
 const tryJSONParse = require('./try-json-parse');
+const isObject = require('lodash.isobject');
```

Comment:
wasn't sure what's better here, if write the `typeof` myself or bring lodash....wdyt?

Reply:
If it's simple enough, I'd prefer doing it ourselves rather than bringing in dependencies.

Reply:
yup, that's what I thought. Will update